### DISCUSSION TITLE
Bunを利用したQiita CLIのインストール方法について

### DISCUSSION THREAD

**Discussion by alpineyahoo** [increments/qiita-discussions#933] [ヘルプ]
Qiita CLI の導入方法について、公式では
```sh
npm install @qiita/qiita-cli --save-dev
npx qiita version # バージョン確認
```
が示されていましたが、これは`node`の代わりのランタイムとして`bun`を利用する場合は
```sh
bun add --dev @qiita/qiita-cli
bunx qiita version
```
と読み違えても問題ありませんでしょうか

**Comment by ohakutsu** [comment]
@alpineyahoo 
Discussionありがとうございます！

Qiita CLIは現在、Node.jsのみのサポートとなっております。
Bunでのご利用を制限しておりませんが、動作保証はできかねます :bow:
